workloads/README.md

```diff
@@ -288,19 +335,29 @@ We target around 100 nodes for this benchmark.
 
 ```yaml
   actions {
-    name: "clique_queries"
+    name: 'clique_queries'
     iterations {
       max_duration_us: 10000000
-      open_loop_interval_ns: 16000000
-      open_loop_interval_distribution: "sync_burst"
+      open_loop_interval_ns: 160000000
```

Comment:
this seems like 1 too many zeros.... did I make a typo?

Reply:
No, my bad, fixed.